### ISSUE TITLE
Look into initiating a bounty via GitHub comments [$5]

### ISSUE THREAD

**Issue by rappo** [bountysource/core#25]
Per suggestion from [this comment](https://github.com/bountysource/frontend/issues/8#issuecomment-17920538) in #8, we should look into checking GitHub comments for a string (like "$+20") and queue up a bounty for that user the next time he visits Bountysource.
##

**Comment by hectorip** [comment]
Is someone working on this yet?

**Comment by PlasmaPower** [comment]
I don't think this is possible given the scale of GitHub and the lack of an API built for this (unless someone knows of one, I couldn't find it).

**Comment by PlasmaPower** [comment]
Although changetip has Github integration IIRC, I think it's built on the mention system. We should look into that. We'd need a user for the bot of course.

**Comment by ayushman1210** [comment]
let me know if this issue still opens  i would like to contribute
